api4/command.go

```diff
@@ -212,9 +212,13 @@ func executeCommand(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// DMs don't have teams, so get the team id from the channel if not specified
```

Comment:
Now that I think about it. Why even bother letting the user specify TeamId? Just do the `commandArgs.TeamId = channel.TeamId` every time.

Reply:
What, if anything, do we do about `CommandArgs` with this change? Do we need to deprecate the `TeamId` field in some way?